src/bootstrap_handler.rs

```diff
@@ -129,8 +121,27 @@ impl BootstrapHandler {
 
     fn write_bootstrap_file(&mut self, mut contacts: Contacts) -> io::Result<()> {
         contacts = contacts.clone().into_iter().unique().collect();
-        self.file_path = BootstrapHandler::get_file_path();
-        let mut file = try!(File::create(&self.file_path));
+        let _ =  fs::create_dir_all(self.file_path.parent().unwrap());
```

Comment:
I would say absolutely, lets not introduce any unwraps (Except for tests) and remove existing ones as well.